### PR TITLE
Document how to enable and set up Prometheus metrics collection

### PR DESCRIPTION
**Pull request by albertodonato**

### CODE REVIEW

File: en/manage-prometheus-metrics.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+- Per request DB queries counts
+
+All available metrics are prefixed with `maas_`, to make it easier to look them
+up in Prometheus and Grafana UIs.
+
+## Enabling Prometheus endpoints
+
+Prometheus endpoints are exposed over HTTP by the `rackd` and `regiond`
+processes under the default `/metrics` path, whenever the
+`python3-prometheus-client` library is installed.
```
Comment (blakerouse) [inline en/manage-prometheus-metrics.md]: Should clarify that with the MAAS snap it woks out of the box.

Probably provide the apt command to run for the packaged version of MAAS.

File: en/manage-prometheus-metrics.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+After installing the library on each MAAS node, the `rackd` and `regiond`
+services must be restarted.
```
Comment (blakerouse) [inline en/manage-prometheus-metrics.md]: Restart commands would be helpful here as well.

File: en/manage-prometheus-metrics.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+      metrics_path: /metrics
+      scheme: http
+      static_configs:
+        - targets:
+          - <maas-IP>:5239  # for regiond
+          - <maas-IP>:5249  # for rackd
+```
+
+If the MAAS installation includes multiple nodes, the `targets` entries must be
+adjusted accordingly, to match services deployed on each node.
```
Comment (blakerouse) [inline en/manage-prometheus-metrics.md]: Should probably update the yaml example to contact different machines, so its more obvious what you mean here.

File: en/manage-prometheus-metrics.md (hunk 2 above)
Comment (albertodonato) [reply en/manage-prometheus-metrics.md]: done

File: en/manage-prometheus-metrics.md (hunk 1 above)
Comment (albertodonato) [reply en/manage-prometheus-metrics.md]: done

File: en/manage-prometheus-metrics.md (hunk 3 above)
Comment (albertodonato) [reply en/manage-prometheus-metrics.md]: done

**Pull request merged**